Docs/landing/data/releases.toml

```diff
@@ -1,7 +1,12 @@
-current = "2.7.0"
```

Comment:
I was expecting this to be 2.7.3 but apparently master is out of date with respect to this.

Reply:
ack